test/unit/test_actions.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+from groundlight import ExperimentalApi
+from openapi_client.exceptions import NotFoundException
+
+
+@pytest.fixture(name="gl")
+def _gl() -> ExperimentalApi:
+    return ExperimentalApi()
+
+
+def test_create_action(gl: ExperimentalApi):
+    det = gl.get_or_create_detector("testing_detector", "test_query")
+    rule = gl.create_rule(det, "test_rule", "EMAIL", "test@example.com")
+    rule2 = gl.get_rule(rule.id)
+    assert rule == rule2
+    gl.delete_rule(rule.id)
+    with pytest.raises(NotFoundException) as _:
+        gl.get_rule(rule.id)
+
+
+def test_get_all_actions(gl: ExperimentalApi):
+    num_test_rules = 103  # needs to be larger than the default page size
```

Comment:
I don't think we should be creating this many rules in unit tests -- instead, we should patch the default page size to something smaller.